src/einsteinpy/coordinates/velocity.py

```diff
@@ -1,7 +1,7 @@
 import astropy.units as u
 import numpy as np
 
-from .core import BoyerLindquist, Cartesian, Spherical
```

Comment:
removing the `.` wouldn't work. It's about absolute n relative imports

Reply:
S what should I do here?

Reply:
just revert to `from .core import ....`

Reply:
Then it doesn't work on my system. I used einsteinpy.coordinates then !